src/indra_cogex/apps/data_display/__init__.py

```diff
@@ -36,12 +36,27 @@
 data_display_blueprint = Blueprint("data_display", __name__)
 
 MORE_EVIDENCES_LIMIT = 10
-curation_cache = CurationCache()
+
+
+def stringify_curations(curation_list: Curations):
+    """Turn pa_hash and source_hash entries to strings for JSs compatibility
+
+    Parameters
+    ----------
+    curation_list :
+        A list of curations to change the type for the source_hash and
+        pa_hash in.
+    """
+    for curation in curation_list:
+        pa_hash = str(curation["pa_hash"])
+        source_hash = str(curation["source_hash"])
+        curation["pa_hash"] = pa_hash
+        curation["source_hash"] = source_hash
```

Comment:
Does it make sense to implement this as an in-place operation?

Reply:
Hmm, yeah maybe it's safer to return instead

Reply:
Resolved in https://github.com/bgyori/indra_cogex/pull/115/commits/e3242250267c6a88728b3d6553e7ea86be20d512